app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+const mongoose = require('mongoose');
+const url = 'mongodb+srv://Immanuel50:chukaglorY55@cluster0-ggn2t.mongodb.net/bucket-list-app?retryWrites=true&w=majority';
+mongoose.connect(url, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+});
+
+
+
+app.get('*', (req, res) => {
+    res.send('<h1>Page Not Found!</h1>')
+});
+
+
+
+app.listen(8080, process.env.IP, () => {
```

Comment:
why do you have this statement `process.env.IP`?

I am expecting `app.listen(8080, () => {`